### DISCUSSION TITLE
[Bug] Perpetual notifications I cannot see

### DISCUSSION THREAD

**Discussion by douglasg14b** [community/community#6925] [Other Feature Feedback, Questions, & Ideas]
I always have the notification icon on the upper right, even if all notifications are read. This started a couple weeks ago. There appears to be 5 notifications I cannot see.

![image](https://user-images.githubusercontent.com/1400380/139308178-fa0bd50a-b855-45d5-9f7f-1341945a1071.png)

**If I clear all notifications:**

![image](https://user-images.githubusercontent.com/1400380/139308301-7add64f3-69b1-440d-aba6-f4755dc15a1a.png)

**Comment by Avivhdr** [comment]
Same here. I tried solving it by logging out and back in and by clearing the browser data. that didn't help.
![Kapture 2024-03-06 at 12 16 18](https://github.com/community/community/assets/23265867/c2ba562e-46e9-4d26-a14b-cf375706aee5)
